### PR TITLE
Tail wagging

### PR DESCRIPTION
**Pull request by Morb0**
## About the PR

Adds new emotion for reptilian to start or stop wagging their tail

## Technical details

- Not use emote system, because it not supports custom system behavior, don't have convenient panel and don't have conditions to make toggable emote
- Use simple `Animated` prefix to get animated marking, looks okay while tails is marking
- Support multiple tail markings animation

## Media

new-tail

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: Morb0, SonicDC
- add: The lizards have gained emotion of tail wagging

### CODE REVIEW

**Comment by LankLTE** [comment]
Don’t have an issue with the feature but something about this feels so cursed

**Comment by DogZeroX** [comment]
The path to the texture says "corvax"

**Comment by LankLTE** [comment]
> The path to the texture says "corvax"

The system / component are also in a corvax folder and use it in their namespace

**Comment by Morb0** [comment]
> > The path to the texture says "corvax"
> 
> The system / component are also in a corvax folder and use it in their namespace

Fixed

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
This should be using generic emoting rather than specifically being just for wagging.

**Comment by UbaserB** [comment]
I like the side view, but not the front view animation.

**Comment by NancokPS2** [comment]
Would any other race get a similar treatment?  
It feels very odd because every mob is basically a static static image, yet now reptilians get an animation solely for their tails.  
This is pretty good, but it is really out of place atm.

**Comment by DogZeroX** [comment]
I don't think all races need wagging, if anything when you animate it.

**Comment by VasilisThePikachu** [comment]
This is the best thing to be created ever since sliced bread, though I have to agree having it as an action is a bit weird. Better for it to just be an emote (like "wags tail" etc)

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

**Comment by Eden077** [comment]
Liznerds rise up

File: Resources/Prototypes/Corvax/Entities/Mobs/Customization/Markings/reptilian.yml
Comment (LankLTE) [inline Resources/Prototypes/Corvax/Entities/Mobs/Customization/Markings/reptilian.yml:1]: This needs to not be in a Corvax folder.

**Review by LankLTE — DISMISSED** [review]
This needs to be handled via emotes, not a sidebar action.

**Comment by LankLTE** [comment]
I also think the animation looks really odd, too smooth and out of place. I’d get the opinions of an art director on that though as it isn’t my strong suit.

**Comment by Morb0** [comment]
> This needs to be handled via emotes, not a sidebar action.

I got permit to not reinventing emote system for tails.
https://discord.com/channels/310555209753690112/560845886263918612/1150679877582856233

**Comment by LankLTE** [comment]
> I got permit to not reinventing emote system for tails.

She had misunderstood and thought the alternative was using the emote wheel PR (that I believe is now closed). After further discussion we agree this should be implemented via chat emotes.

**Review by moonheart08 — CHANGES_REQUESTED** [review]
The spritework is awful, needs redone by an animator who understands what weight is. As of now the animation is extremely stiff.

**Comment by SonicHDC** [comment]
Downloadedpng
Я полагаю, что как чёрные кормашки или мелкие вставки подойдёт.
Или же сам цвет вставок сделать тусклей/светлей.

**Comment by SonicHDC** [comment]
Multitabs PRs answers. XD

**Comment by SonicHDC** [comment]
When?

**Comment by metalgearsloth** [comment]
@Morb0 add prediction and I'll merge

**Comment by UbaserB** [comment]
You will need to include this tail marking for wagging now #23551

**Review by metalgearsloth — APPROVED** [review]

**Comment by Hmeister-fake** [comment]
I am so happy

**Comment by SonicHDC** [comment]
Yey!

**Comment by Aexxie** [comment]
hell yeah now I dont need to do it myself

**Pull request merged**